server/routes/webhook.py

```
@@ -0,0 +1,46 @@
+import hashlib
+
+from fastapi import APIRouter,Request,HTTPException
+from redis import Redis
+from rq import Queue
+from worker.main import process_pr
+router=APIRouter()
+
+connection=Redis(host="localhost",port=6379,db=0)
+queue=Queue("github_prs",connection=connection)
+
+WEBHOOK_SECRET="hemanth"
```

Ruff (0.13.2)

Possible hardcoded password assigned to: "WEBHOOK_SECRET"

(S105)